server/bin/pbench-server-activate-create-results-dir-structure

```diff
@@ -90,14 +60,19 @@ typeset -i nerrs=0
 
 # it's OK if the directories exist already.
 mkdir -p ${archive_dir}
+
+pbench_incoming_dir=${pbench_dir}/public_html/incoming
+pbench_results_dir=${pbench_dir}/public_html/results
```

Comment:
It would probably be good to consider continuing the DRY principle and define a variable for the `public_html` path rather than repeating that each time.

Reply:
Theoretically, yes, but this script is going away soon. This PR is just an intermediate step to the ansible playbooks and, as a consequence, obsolescence for all the activate scripts (with the create_crontab exception),

Reply:
That would be great, but I still see these variables in #1199.

Reply:
See above for a possible explanation. If that's not correct, can you point out where you see them?

Reply:
Yes, my comment above was a result of not noticing that PR #1199 contains these exact same commits.